docs/migrations/v7-to-v7_1.md

```diff
@@ -0,0 +1,74 @@
+# Migrating from v7 to v7.1
+
+This guide provides instructions for migrating to version `v7.1.0` of ibc-go.
+
+There are four sections based on the four potential user groups of this document:
+
+- [Migrating from v7 to v7.1](#migrating-from-v7-to-v71)
+  - [Chains](#chains)
+  - [IBC Apps](#ibc-apps)
+  - [Relayers](#relayers)
+  - [IBC Light Clients](#ibc-light-clients)
+
+**Note:** ibc-go supports golang semantic versioning and therefore all imports must be updated on major version releases.
+
+## Chains
+
+In the previous release of ibc-go, the localhost `v1` light client module was deprecated and removed. The ibc-go `v7.1.0` release introduces `v2` of the 09-localhost light client module.
+
+Chain developers must register the 09-localhost `AppModuleBasic` in `app.go`.
+
+```go
+import (
+  // ...
+  localhost "github.com/cosmos/ibc-go/v7/modules/light-clients/09-localhost"
+)
+// ...
+ModuleBasics = module.NewBasicManager(
+  ...
+  ibc.AppModuleBasic{},
+  localhost.AppModuleBasic{},
+  ...
+)
+```
```

Comment:
I think making this a diff is nice. What do you think?

Reply:
I can change it to a diff, I don't feel super strong about it. I preferred using the diff as well but I noticed it doesn't render in the docs website theme as well as it does in regular markdown.

Reply:
Our release notes link to github, so I think it could still be useful